ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -392,7 +395,8 @@ public void close() {
           TimeUnit.MILLISECONDS);
       indexSegmentCount++;
       if (indexSegmentCount == 1 || indexSegmentCount % 10 == 0) {
-        logger.info("Index segment processing (on-demand scanning) for compaction stats complete for {}", indexSegment);
+        logger.info("CompactionStats: Index segment {} processing complete (on-demand scanning) for store {}",
```

Comment:
minor: The other one is named "Container Stats" (with a space). Good to be consistent (either have the space or don't)